Superkunair/maiaProject
# Maia02 db connection - Created the mongoDB connection

Steps to test: 
Locate in to the server folder and then do the following things
1- Go to the file "start.js"
2-Uncomment the line of the database connection in the server startup
3-rename the file called ".env.template" to ".env"
4-Add the enviroment variables to do the DatabaseConnection (Ask me privately to get the values)
5- locate in the principal server folder an then do a `npm run debug`
6- you should see the listed databases
